pkg/smartcontract/param_type.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"io/ioutil"
```

Comment:
It's deprecated, I'm not sure Go 1.13 already has all required substitutes for it, but if it does we better use them to avoid the need of replacing it in future.

Reply:
It just will be moved to `os` package in Go 1.16 (see https://golang.org/src/io/ioutil/ioutil.go?s=1167:1213#L26). In Go 1.13 there's no substitution for that yet. Can we leave this as is?

Reply:
Sure, if it's not there yet in 1.13 then it's OK to have `ioutil`. It's just that some functionality has been moved earlier and if we're dealing with that we should be using new interfaces.